JM/CF/Scripts/3_Game/CommunityFramework/RPC/RPCManager.c

```diff
@@ -356,12 +356,21 @@ class RPCManager
 	{
 		if ( !m_RPCActions.Contains( modName ) )
 		{
-			//GetLogger().Log( "Creating RPC mod " + modName, m_UpdateChecker );
+			Print("Creating RPC mod " + modName);
 			m_RPCActions.Set( modName, new ref map< string, ref RPCMetaWrapper > );
 		}
 		
-		//GetLogger().Log( "Creating RPC function " + modName + "::" + funcName, m_UpdateChecker );
-		auto wrapper = new ref RPCMetaWrapper( instance, singlePlayerExecType );
+		Print("Adding RPC function " + modName + "::" + funcName);
```

Comment:
No info log on each successful normally expected action until we have log levels.
Comment the line out